src/Utils/localImageHelpers.ts

```diff
@@ -88,9 +88,8 @@ export const useLocalImageStorage = (
     )
 
   const changeLocalImage = async () => {
-    setLocalImage(null)
-
     if (isImageAvailable || !key) {
+      setLocalImage(null)
```

Comment:
Before this correction, the image flickered because it was set to `null` during the loading time.